src/AccessibilityInsights.SharedUx/Properties/Resources.resx

```diff
@@ -1561,11 +1558,32 @@ Do you want to change the channel? </value>
   <data name="ChangeChannelContainedDialog_BtnExit" xml:space="preserve">
     <value>Do not change channel</value>
   </data>
-<data name="ChangeChannelContainedDialog_BtnOk" xml:space="preserve">
+  <data name="ChangeChannelContainedDialog_BtnOk" xml:space="preserve">
     <value>Change channel</value>
   </data>
   <data name="ScreenshotWindowTitle" xml:space="preserve">
     <value>Accessibility Insights for Windows - Screenshot</value>
     <comment>Title for window containing screenshot</comment>
   </data>
-</root>
+  <data name="LabelUIAccessAvailable" xml:space="preserve">
+    <value>UIAccess is available</value>
+  </data>
+  <data name="LabelUIAccessNotAvailable" xml:space="preserve">
+    <value>UIAccess is not available</value>
+  </data>
+  <data name="hlVersion" xml:space="preserve">
+    <value>Version</value>
+  </data>
+  <data name="Placeholder" xml:space="preserve">
+    <value>*** placeholder ***</value>
+    <comment>Gets replaced at runtime</comment>
+  </data>
+  <data name="VersionLinkContentFormat" xml:space="preserve">
+    <value>Version {0}</value>
```

Comment:
>{0} [](start = 19, length = 3)

This is wrong. This string strips out the leading zero's in the last 2 pieces of the Uri, and those zero's are a required pat of the Uri